includes/core/class-member-directory.php

```diff
@@ -1596,13 +1596,12 @@ function filters( $directory_data ) {
 								case 'text':
 
 									$value = stripslashes( $value );
-
 									$field_query = array(
 										'relation' => 'OR',
 										array(
 											'key'       => $field,
 											'value'     => trim( $value ),
-											'compare'   => 'LIKE',
+											'compare'   => apply_filters( 'um_members_directory_filter_text', '=', $field )
```

Comment:
Check the reason for using '=' by default instead of 'LIKE'
maybe we need '=' only for some selected fields?

Old logic - filter by the text field makes the search by Like